docs/community/contributing.md

```diff
@@ -0,0 +1,69 @@
+---
+id: contributing
+title: Contributing
+sidebar_label: Contributing
+---
+
+We are thrilled that you are considering contributing to Veramo. Your contributions are incredibly valuable to us, and we appreciate your interest in helping us grow and improve.
+
+As a new contributor, you may be wondering why your contributions matter. The truth is, every open-source project relies on community contributions to thrive. Each contribution, no matter how small, helps move the project forward and improves its functionality. Your unique perspective and ideas can bring fresh insights and new solutions to Veramo, helping it to evolve and meet the needs of our community.
+
+Contributing to our project will give you the opportunity to work with other talented individuals, learn new technologies, and collaborate on real-world problems. 
+So, whether you're a seasoned developer or just starting, we welcome your contributions to our project. We're excited to see what ideas you'll bring to the table and how you'll help us make Veramo even better. Thank you for considering contributing, and we can't wait to work with you!
+
+### How to contribute
+As a developer, here are some ways to get involved
+* [Report a bug](#reporting-a-bug): You can help the community to identify and fix bugs quickly by reporting any issues you have with the code.
+* [Discuss the current state of the code](https://github.com/uport-project/veramo/discussions): Join in on discussions about the state of the code. Participate in discussions and help us better understand what the project needs in order to make decisions that are in the best interest of everyone involved.
+* [Submit a fix](#submitting-a-fix): Submitting a fix is a collaborative effort, and we’d like for you to contact the maintainers of this project before carrying on with fixes. 
+* Propose new features: When proposing a new feature, please provide as much information and detail as possible. This includes a clear description of the feature, an explanation of why it is needed and its benefits, as well as any existing code or resources that can be used to implement the feature. 
+* Answer questions: Help both old and new community members to understand the project and get started quickly by answering questions about the project. Discussions happen in our [GitHub discussion](https://github.com/uport-project/veramo/discussions) and [Discord community](http://discord.gg/sYBUXpACh4).
+* Submit an issue in the docs: If you come across an issue or a bug in the project's documentation, you can submit an issue report to bring it to the attention of the project maintainers.
+* Suggest edits to the docs: This could involve correcting errors, improving clarity, or adding missing information. You can submit your suggestions through a pull request or by directly contacting the project maintainers. Make sure to explain why you're suggesting the changes and provide examples or references to support your suggestions.
```

Comment:
```suggestion
* Suggest edits to the docs: This could involve correcting errors, improving clarity, or adding missing information. You can submit your suggestions through a [pull request](https://github.com/uport-project/veramo-website/fork) or by directly contacting the project maintainers. Make sure to explain why you're suggesting the changes and provide examples or references to support your suggestions.
```